<<<<<<< HEAD
#Service to Ping the robot
---
int64 ret
string msg
=======
#Service to Ping the robot
---
int64 ret
string msg
>>>>>>> 1b1f87b6247333221ca1660d810eda01b82fd03f
